dozer/cogs/starboard.py

```diff
@@ -0,0 +1,366 @@
+                        fetch_message(db_msgs[0].starboard_message_id)
+                except discord.NotFound:
+                    DOZER_LOGGER.warning(f"Cannot find Starboard Message {db_msgs[0].starboard_message_id} to remove")
+                    starboard_msg = discord.Object(db_msgs[0].starboard_message_id)
+                await self.remove_from_starboard(config, starboard_msg, True)
+                return
+
+    @Cog.listener()
+    async def on_reaction_add(self, reaction, user):
+        """Pass the reaction add event onto our handler"""
+        await self.starboard_check(reaction, user)
+
+    @Cog.listener()
+    async def on_reaction_remove(self, reaction, user):
+        """Pass the reaction remove event onto our handler"""
+        await self.starboard_check(reaction, user)
+
+    @guild_only()
+    @group(invoke_without_command=True, aliases=['hof'])
+    async def starboard(self, ctx):
+        """Show the current server's starboard configuration.
+         A starboard (or a hall of fame) is a channel the bot will repost messages in if they receive a certain number\
+         of configured reactions.
+
+         To configure a starboard, use the `starboard config` subcommand.
+         """
+        config = await StarboardConfig.get_by(guild_id=ctx.guild.id)
+
+        if config:
+            await ctx.send(embed=self.make_config_embed(ctx, f"Starboard configuration for {ctx.guild}", config[0]))
+        else:
+            await ctx.send(f"This server does not have a starboard configured! See `{ctx.prefix}help starboard` for "
+                           f"more information.")
+
+    starboard.example_usage = """
+    `{prefix}starboard` - Get the current starboard settings for this server
+    """
+
+    @has_permissions(manage_guild=True, manage_channels=True)
+    @bot_has_permissions(add_reactions=True, embed_links=True)
+    @starboard.command()
+    async def config(self, ctx, channel: discord.TextChannel, star_emoji: typing.Union[discord.Emoji, str],
+                     threshold: int,
+                     cancel_emoji: typing.Union[discord.Emoji, str] = None):
+        """Modify the settings for this server's starboard"""
+        for emoji in [emoji for emoji in [star_emoji, cancel_emoji] if emoji is not None]:
+            try:
+                # try adding it to make sure it's a real emoji. This covers both custom emoijs & unicode emojis
+                await ctx.message.add_reaction(emoji)
+                await ctx.message.remove_reaction(emoji, ctx.guild.me)
+            except discord.HTTPException:
+                await ctx.send(f"{ctx.author.mention}, bad argument: '{emoji}' is not an emoji, or isn't from a server "
+                               f"{ctx.me.name} is in.")
+                return
+
+        config = StarboardConfig(guild_id=ctx.guild.id, channel_id=channel.id, star_emoji=str(star_emoji),
+                                 threshold=threshold, cancel_emoji=str(cancel_emoji))
```

Comment:
If `cancel_emoji` is `None`, this inserts the string `'None'` instead of an SQL `NULL`. I don't think this will cause any noticeable problems - the string `'None'` isn't a reaction that could appear on a message, and it actually serves as a workaround for #195 - but it's something worth revisiting to make it a proper `NULL` after that issue is closed.